src/sections/managedTeams.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Managet teams</title>
+  </head>
+
+  <body>
+    <section class="2xl:grid 2xl:justify-center 2xl:pr-96">
+      <div class="overflow-x-hidden pb-12">
+        <div class="flex flex-col gap-2 lg:mt-40 gmax-w-5xl w-3/4 m-6 xl:ml-44">
+          <p class="text-gray-400 font-railwayBold">Managed Teams</p>
+          <h2 class="font-railwayBold text-3xl">
+            Skalieren Sie langfristig Ihre <br />
+            <span class="text-[#116355]">Softwareentwicklungsteams</span> <br />
+            ohne langwieriges Recruitment
+          </h2>
+          <p>
+            kurz beschreiben was und für wen Managed Teams sind anbieten ipsum
+            dolor sit amet <br />, consetetur sadipscing elitr, sed diam nonumy
+            eirmod tempor invidunt ut labore et dolore <br />
+            magna aliquyam erat, sed diam voluptua. At vero eos et accusam et
+            justo duo dolores et ea <br />
+            rebum. Stet clita kasd gubergren, no sea takimata sanctus est L
+          </p>
+        </div>
+        <div class="mt-4 m-6 xl:ml-44">
+          <form
+            action=""
+            method="post"
+            class="flex flex-col xl:flex-row font-railwayBold"
+          >
```

Comment:
I don't really understand why you decided to use the form like this, especially since I can't find the input tag it's supposed to go with! If you have a good reason, please tell me. If not, you could just use a different tag. Forms are mainly used for working with forms and putting input tags together.